frontend/app/components/Dashboard/Widgets/CustomMetricsWidgets/ProgressBarChart.tsx

```
@@ -0,0 +1,84 @@
+  const formattedNumber = (num: number) => {
+    return Intl.NumberFormat().format(num);
+  }
+
+  // we mix 1 original, then 1 comparison, etc
+  const mergedNameMap: { data: any, isComp: boolean, index: number }[] = [];
+  for (let i = 0; i < data.namesMap.length; i++) {
+    if (!data.namesMap[i]) {
+      continue;
+    }
+    mergedNameMap.push({ data: data.namesMap[i], isComp: false, index: i });
+    if (compData && compData.namesMap[i]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)